crates/hyperswitch_connectors/src/connectors/deutschebank/transformers.rs

```diff
@@ -162,6 +231,55 @@ impl TryFrom<&DeutschebankRouterData<&PaymentsAuthorizeRouterData>>
                                 last_name: billing_address.get_last_name()?.clone(),
                             }))
                         }
+                        PaymentMethodData::Card(ccard) => {
```

Comment:
In this case, the card payment request will only be constructed during a CIT (when customer acceptance is passed). Normal payments do not require customer acceptance. Please refactor the try_from implementation to support normal card payments as well.